src/notification-toast/notification-toast.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from "react";
+import { NotificationToastProps } from "./types";
+import {
+    Container,
+    Content,
+    Description,
+    StyledIconButton,
+    Title,
+    Wrapper,
+} from "./notification-toast.styles";
+import { Icon } from "src/icon";
+
+export const Notification = ({
+    type,
+    title,
+    children,
+    icon,
+}: NotificationToastProps): JSX.Element => {
+    const [isVisible, setVisible] = useState<boolean>(true);
+
+    const handleDismiss = () => {
+        setVisible(false);
+    };
+
+    if (!isVisible) return null;
+
+    return (
+        <Wrapper $type={type}>
+            <Container>
+                <Content>
+                    <StyledIconButton $type={type} onClick={handleDismiss}>
+                        <Icon type="cross" />
```

Comment:
This component will be deprecated. Please use `CrossIcon` from `import { CrossIcon } from "@lifesg/react-icons/cross";`

Refer to https://designsystem.life.gov.sg/reacticons/index.html?path=/docs/icon--collection